libdnf/dnf-context.cpp

```diff
@@ -2539,23 +2572,23 @@ dnf_context_load_vars(DnfContext * context)
     priv->varsCached = true;
 }
 
-static libdnf::ConfigMain cfgMain;
+static libdnf::ConfigMain globalMainConfig;
```

Comment:
I think that renaming of cfgMain belongs into the first commit. This commit is about "Add possibility to store best configuration in context part."